package.json

```diff
@@ -22,7 +22,7 @@
     "colors-tmpl": "~0.1.0",
     "core-util-is": "~1.0.1",
     "cpr": "~0.3.2",
-    "getos": "~1.0.1",
+    "getos": "^2.3.0",
```

Comment:
Sorry.
Didn't know this was intentional. npm/npm#4589
I should have sticked to ~ anyway.